buildSrc/src/main/kotlin/Libs.kt

```diff
@@ -15,7 +15,7 @@ object Libs {
    }
 
    object Kotlinx {
-      private const val version = "1.5.0"
```

Comment:
From your other comment:
> It requires all of our users to update their codebase to kotlin 1.9

Here we used version `1.5.0` that were using kotlin 1.8 ([see the release notes](https://github.com/Kotlin/kotlinx.serialization/releases/tag/v1.5.0))